src/Forms/Form.php

```diff
@@ -521,7 +521,7 @@ public function setFieldMessage(
         return $this;
     }
 
-    public function castingHelper($field, bool $useFallback = true)
+    public function castingHelper(string $field, bool $useFallback = true): ?string
```

Comment:
These are all nullable because `ViewableData::castingHelper()` returns null, and most if not all implementations of this method call `parent::castingHelper()`